src/item.cpp

```diff
@@ -7050,15 +7050,17 @@ void item::process_temperature_rot( int temp, float insulation, const tripoint p
 
     // process temperature and rot at most once every 100_turns (10 min)
     // note we're also gated by item::processing_speed
-    time_duration smallest_interval = 100_turns;
+    time_duration smallest_interval = 10_minutes;
     if( now - last_temp_check < smallest_interval ) {
         // Could be newly created item.
         if( specific_energy < 0 ) {
             if( carried ) {
                 temp += 5; // body heat increases inventory temperature
             }
             calc_temp( temp, insulation, now );
-            calc_rot( now );
+            if( goes_bad() ) {
```

Comment:
How about instead exiting calc_rot() immediately if goes_bad() == false?

Reply:
Yeah that works too and there would be less if-statements.